### PR TITLE
Example of RStudio Rmd to train-tune-deploy

### PR DESCRIPTION
**Pull request by georschi**
Example of RStudio Rmd to train-tune-deploy. 
The example uses the widely used Abalone dataset. Uses R to visualise and preprocess the data and then uses SageMaker built-in XGBoost algorithm to train, tune and deploy a model.

### CODE REVIEW

File: r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+<br><br>
+
+## Download Data
+
+For this example we will be using the famous abalone dataset as can be found on the [UCI dataset archive](https://archive.ics.uci.edu/ml/datasets/Abalone) where we will create a model to predict the age of an abalone shell based on physical measurements. 
+
+
+
+```{r}
+data_file <- 'http://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data'
```
Comment (jkroll-aws) [inline r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd:57]: Can you use the AWS-hosted version which is available in S3?
`s3://sagemaker-sample-files/datasets/tabular/uci_abalone/abalone.csv`

**Comment by jkroll-aws** [comment]
What is the HTML file? It seems very large, I wonder if it contains embedded images that can be added as standalone relative PNG files. Why is the notebook itself not sufficient?

**Comment by atqy** [comment]
Looking at the code - in accordance with the rest of the codebase, can you convert main.R to a R Jupyter notebook? I think it would be more beneficial to customers if you include an explanation of the code and they can see the output of various lines of code.

Also, we want to test this code daily to make sure that it works. It only tests notebooks, so this is another benefit to having the main code in a Jupyter notebook

File: r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+role_arn <- sagemaker$get_execution_role()
+
+
+```
+
+<br><br>
+
+## Download Data
+
+For this example we will be using the famous abalone dataset as can be found on the [UCI dataset archive](https://archive.ics.uci.edu/ml/datasets/Abalone) where we will create a model to predict the age of an abalone shell based on physical measurements.
```
[the remark this answers is not in the document]
Comment (georschi) [reply r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd:52]: thanks!

File: r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd (hunk 1 above)
Comment (georschi) [reply r_examples/rstudio-train-tune-deploy-xgboost/abalone-train-tune-deploy-xgboost-model.Rmd:57]: thanks - updated this

**Comment by georschi** [comment]
@jkroll-aws good point on the HTML, instead I am rendering it in .md and will add to the commit. The reason for adding it is only for readability reasons. The HTML file contains a run version of the notebook so the reader can see how the notebook run and what outputs to expect.

**Comment by georschi** [comment]
For readability purposes I changed the output from HTML to markdown so that it can also render nicely on github.com 

Unfortunately RStudio does not support R Jupyter Notebooks so converting the example in such a file would not be an option at this point.

**Review by jkroll-aws — APPROVED** [review]

**Review by atqy — APPROVED** [review]

**Comment by zhaoqizqwang** [comment]
Closing this PR since it was created a long time ago and didn't pass all the checks. Feel free to open a new pull request instead.

**Pull request closed without merging**